src/atoms/RadioButton/Backdrop.native.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components/native';
+import View from '../View';
+
+const Backdrop = styled(View)(
+  (props) => `width: ${props.width};
+height: ${props.height};
+border-radius: ${props.borderRadius};
+background-color: ${props.backgroundColor};
+`,
+);
```

Comment:
```suggestion
const Backdrop = styled(View)(
  (props) =>
    `
    width: ${props.width};
    height: ${props.height};
    border-radius: ${props.borderRadius};
    background-color: ${props.backgroundColor};
    `,
);
```

I don't think prettier works in these situations. Let's just ensure we format it properly